src/component/toolbox/feature/MagicType.ts

```diff
@@ -153,11 +155,13 @@ class MagicType extends ToolboxFeature<ToolboxMagicTypeFeatureOption> {
         // Change title of stack
         if (type === 'stack') {
             const seriesOptions = newOption.series as (SeriesOption & { stack: string })[];
+            const titldLang = ecModel.getWithLocale(['toolbox', 'magicTypeLang', 'title', 'tiled']);
```

Comment:
It seems that this word `titldLang` is a typo, do you mean `tiledLang` or `tiledTitleLang`?